src/test/lib/operators/operator_performance_data_test.cpp

```diff
@@ -107,7 +112,8 @@ TEST_F(OperatorPerformanceDataTest, TableScanPerformanceData) {
     const auto& performance_data = dynamic_cast<TableScan::PerformanceData&>(*table_scan->performance_data);
     EXPECT_GT(performance_data.walltime.count(), 0ul);
     EXPECT_EQ(performance_data.num_chunks_with_early_out, 0ul);
-    EXPECT_EQ(performance_data.num_chunks_with_binary_search, 2ul);
+    EXPECT_EQ(performance_data.num_chunks_with_all_rows_matching, 0ul);
```

Comment:
So we don't have a short cut for chunk 1 with all 2's and the predicate `=2`?

Reply:
Exactly, as far as I can tell, there is no such shortcut for ColumnVsValue scans in the sorted segment search, only for between scans. For Equals scans, we directly call `_get_first/last_bound()`, which I consider to be the search operation because after that we only write the output.

https://github.com/hyrise/hyrise/blob/e3bb51cb6b93503eb3dcca5411a36344b2b226f9/src/lib/operators/table_scan/sorted_segment_search.hpp#L80

Maybe you could consider some parts of the NotEquals ColumnVsValue scans as shortcuts, but those are not included in the tests